src/alignment/State.cc

```diff
@@ -60,67 +64,62 @@ void state::updateSelectability(int i){
     }
 } 
 
+bool state::consistent(Edge& e, Edge& f){
+   if ((e.first->j <= f.first->j && e.second->j >= f.second->j)
+       || (e.first->j >= f.first->j && e.second->j <= f.second->j)) {
+       return false;
+   } else if (e.first->j == f.first->j && e.second->j == f.second->j) {
```

Comment:
This is a bug. This condition to check whether e and f are the same edges should be the first condition. As it is, this condition is never true. Please, also give a short comment on this condition, like `// edges e and f are equal`.

Reply:
done